src/VisualStudio/Core/Def/HACK_ThemeColorFixer.cs

```diff
@@ -16,12 +21,16 @@ namespace Microsoft.VisualStudio.LanguageServices
     /// from its default blue. As a work around, we listen to <see cref="IClassificationFormatMap.ClassificationFormatMappingChanged"/>
     /// and update the classification format maps that we care about.
     /// </summary>
-    [Export]
```

Comment:
It appears this exact type is also being consumed [here](https://github.com/dotnet/roslyn/blob/be3834ceddde9ac0ceb5921cddfc51f46965d6a2/src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs#L48) which is being caught by the failing integration tests.